packages/plugin-ext/src/common/plugin-protocol.ts

```diff
@@ -442,6 +442,7 @@ export interface PluginModel {
         version: string;
     };
     entryPoint: PluginEntryPoint;
+    packageUri: string;
     packagePath: string;
```

Comment:
@owlJaeger please see here how API should be deprecated https://github.com/eclipse-theia/theia/blob/master/doc/api-management.md#deprecation

In this case you need only `@deprecated` tag, no `@since` or `@experimental`. You can read the whole document to learn why.

Reply:
Done